operator/scripts/retry.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# shellcheck source=./hack/common.sh
+source "$(dirname "$0")/../hack/common.sh"
+
+eecho() {
+  echo "$@" >&2
+}
+
+die() {
+  eecho "$@"
+  exit 1
+}
+
+function main() {
+    if [ $# -lt 2 ]; then
+        die "Usage: $0 <number of attempts> <command> [ <command arg> ... ]"
```

Comment:
I might be missing something but the argument processing below seems to require two arguements.
In that case also the invocation in the circleci config is wrong...

Reply:
changed, thanks